### ISSUE TITLE
Terminals with same path can't be saved as default terminal in settings

### ISSUE THREAD

**Issue by Myuki** [files-community/Files#1441]
**Describe the bug**
If have 2 terminals with same path in terminal.json, the second terminal can't be saved as default terminal in settings.

**To Reproduce**
1.  Config 2 terminals with same path in terminal.json like that
```json
{
  "version": 1,
  "defaultTerminalPath": "wt.exe",
  "terminals": [
    {
      "name": "Windows Terminal",
      "path": "wt.exe",
      "arguments": "-d .",
      "icon": ""
    },
    {
      "name": "WSL",
      "path": "wt.exe",
      "arguments": "-d . wsl.exe",
      "icon": ""
    }
  ]
}
```
2. Go to 'Settings -> Preference'.
3. Set 'Terminals' to 'WSL'.
3. Leave settings.
4. Go to 'Settings -> Preference'.
5. The 'Terminals' became 'Windows Terminal' and can't be saved as 'WSL'.

**Expected behavior**
The 'Terminals' should be 'WSL'.

**Desktop (please complete the following information):**
 - OS Version: [Windows 10 2004 19041.329]
 - App version: [v0.11.0]

**Comment by soumyamahunt** [comment]
@yair100 we should save name of the terminal instead of path for default terminal.

**Comment by yaichenbaum** [comment]
@Myuki This issue should be resolved with #2764.
